crates/octez/src/async/client.rs

```diff
@@ -19,14 +20,19 @@ const DEFAULT_BINARY_PATH: &str = "octez-client";
 
 type StdOut = String;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize)]
 pub struct OctezClientConfig {
     binary_path: PathBuf,
+    #[serde(skip_serializing_if = "skip_serializing_base_dir")]
```

Comment:
I think useful to see the exact temp dir that the client uses to quickly inspect baker logs, keys, highwatermarks etc. mostly for advanced users

Reply:
Updated. octez node config needs to be updated later on to be consistent.